session_settings.go

```diff
@@ -124,9 +124,9 @@ func (s SessionSettings) BoolSetting(setting string) (bool, error) {
 	}
 
 	switch stringVal {
-	case "Y", "y":
+	case "Y", "y", "true", "True":
```

Comment:
I would avoid this as these configs are common between every quickfix client in other languages so better to keep consistency when possible